scripts/brp-fix-desktop-files

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Fix .desktop files to be compliant with XDG specification
```

Comment:
Why? .desktop files issuee corrected by this script is quite widespread and goes often unnoticed.

Trivial to automatically fix with script, tedious to manually pursue fixing...